po/fr.po

```diff
@@ -709,9 +703,9 @@ msgid "Already on first feed."
 msgstr "Vous êtes déjà au premier fil."
 
 #: src/feedlistformaction.cpp:410
-#, fuzzy
 msgid "Do you really want to mark all feeds as read (y:Yes n:No)? "
-msgstr "Souhaitez-vous réellement écraser `%s' (y : Oui n : No)? "
+msgstr "Souhaitez-vous réellement marquer tout les fils comme lus "
+"`%s' (o : Oui n : No)?"
```

Comment:
The original string had a space at the end, please add it to the translation as well.